### ISSUE TITLE
Using a prop name of "value" causes an error

### ISSUE THREAD

**Issue by fis-dan-wnuk** [swimlane/ngx-datatable#1048]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
If I use a value of "value" for the prop parameter an error is triggered. This  was working fine before I upgraded from version 9.3.1 to 10.2.3.

**Expected behavior**

The table would display the data properly including all rows of data.

**Reproduction of the problem**

I am able to reproduce very simply:

```
<ngx-datatable class="material striped"
                    [columnMode]="'force'"
                    [rows]="rows"
                    [columns]="columns">
</ngx-datatable>
```

```
export class HomeView {
  rows: any[] = [
    {label: 'Label #1', value: 'value1'}, {label: 'Label #2', value: 'value2'}
  ];
  
  columns: any[] = [
    {name: "Label", prop: 'label'},{name: "Value", prop: 'value'}
  ];
}
```

If I change "value" to "val" the table displays as expected. I would submit a plunker but the live preview is being blocked on my work machine - I can add one later if needed.

**What is the motivation / use case for changing the behavior?**

The name of the key in your object should not impact the datatable component.

**Please tell us about your environment:**

* **Table version:** 10.2.3

* **Angular version:** 4.3.6

* **Browser:** all

* **Language:** all

**Comment by QuentinFchx** [comment]
Duplicate of #993

**Comment by fis-dan-wnuk** [comment]
My apologies - should have looked first for an issue. Closing mine.
